app/grandchallenge/verifications/admin.py

```diff
@@ -17,9 +17,10 @@
     permissions=("change",),
 )
 def mark_verified(modeladmin, request, queryset):
-    queryset.filter(email_is_verified=True).update(
-        is_verified=True, verified_at=now()
-    )
+    for verfication in queryset:
+        verfication.is_verified = True
+        verfication.verified_at = now()
+        verfication.save()
```

Comment:
There is a typo here. `verfication` --> `verification`

Reply:
Fixed